### PR TITLE
replaced container-list ep with pubtools-sign-cosign-signature-list

### PR DESCRIPTION
**Pull request by midnightercz**

### CODE REVIEW

File: pubtools/_quay/signer_wrapper.py [hunk 1]
```diff
@@ -468,9 +468,9 @@ def _list_signatures(self, repository: str, tag: str) -> List[Tuple[str, str]]:
             + f":{tag}"
         )
         existing_signatures = run_entrypoint(
-            ("pubtools-sign", "modules", "pubtools-sign-cosign-container-list"),
-            "pubtools-sign-cosign-container-list",
-            [full_reference],
+            ("pubtools-sign", "modules", "pubtools-sign-cosign-signature-list"),
+            "pubtools-sign-cosign-signature-list",
+            [cast(str, self.config_file), full_reference],
```
Comment (emilyzheng) [inline pubtools/_quay/signer_wrapper.py:473]: http://10.0.150.153:32769/pub/task/426/log/stdout.log
2024-02-26 08:07:45 +0000 [ERROR   ] cosign_list_existing_signatures() missing 1 required positional argument: 'reference'

File: pubtools/_quay/utils/misc.py [hunk 2]
```diff
@@ -162,7 +165,8 @@ def setup_entry_point_cli(
             os.environ[key] = environ_vars[key]
         entry_point_func = pkg_resources.load_entry_point(*entry_tuple)
         if args:
-            func_args = [name]
+            if name:
+                func_args = [name]
             func_args.extend(args)
```
Comment (emilyzheng) [inline pubtools/_quay/utils/misc.py:170]: func_args is not defined before extend when name is None.

File: pubtools/_quay/utils/misc.py [hunk 3]
```diff
@@ -162,7 +165,8 @@ def setup_entry_point_cli(
...
             yield functools.partial(entry_point_func, func_args)
```
Comment (emilyzheng) [inline pubtools/_quay/utils/misc.py:171]: Not adding name is not enough to make pubtools-sign-cosign-signature-list work. I also changed this to:
```
            if entry_tuple[1] == "console_scripts":
                yield functools.partial(entry_point_func, func_args)
            else:
                yield functools.partial(entry_point_func, *func_args)
```

Comment (midnightercz) [reply pubtools/_quay/utils/misc.py:173]: ah yeah, makes sense

**Review by emilyzheng — APPROVED** [review]

**Pull request merged**